completers/git_completer/cmd/root.go

```diff
@@ -1,7 +1,12 @@
 package cmd
 
 import (
+	"fmt"
+	"strings"
+
+	"github.com/google/shlex"
```

Comment:
This is a new dependency, since the aliases are plain strings I needed something to split them into arguments. Do we already have something for this? Or is there a preferred library to use instead?

Reply:
Been avoiding external dependencies as much as possible. Considered `shlex` mutliple times already though due to the issue with quotes, so i'm fine with using that.